### DISCUSSION TITLE
Is Wyze Cameras as motion sensors possible or not anymore?

### DISCUSSION THREAD

**Discussion by jrbecart** [SecKatie/ha-wyzeapi#417] [General]
Since it still mentionned on the front page.
Just asking as it's not 100% clear for me.

Thank you.

**Comment by SecKatie** [comment]
Unfortunately, it is not possible. We just cannot poll fast enough because of their restrictions on the APIs.

Thanks,

--
Josh Mulliken ***@***.***>
Public key: https://keyoxide.org/hkp/joshua%40mulliken.net

**Reply by devinslick** [reply]
Well, I wouldn't say it's impossible.  It's just not very clean.   This is how I'm doing it.   https://github.com/gtxaspec/wz_mini_hacks/discussions/323
